tests/coverage/coverage.cpp

```diff
@@ -756,6 +756,88 @@ TEST(Importer, notMathMLMathNodesInComponentMath)
     EXPECT_EQ("Units reference 'daves' in units 'bobs' is not a valid reference to a local units or a standard unit type.", validator->error(1)->description());
 }
 
+TEST(CoverageValidator, degreeElementWithOneSibling)
+{
+    const std::string math =
+        "<math xmlns=\"http://www.w3.org/1998/Math/MathML\">\n"
+        "  <apply>\n"
+        "    <root/>\n"
+        "    <degree><ci>n</ci></degree>\n"
+        "  </apply>\n"
+        "</math>\n";
+
+    auto parser = libcellml::Parser::create();
+    auto model = libcellml::Model::create("degree");
+
+    auto component = libcellml::Component::create("c");
+    auto variable = libcellml::Variable::create("n");
+    variable->setUnits("second");
+    component->addVariable(variable);
+    component->appendMath(math);
+
+    model->addComponent(component);
+
+    auto validator = libcellml::Validator::create();
+
+    validator->validateModel(model);
+    EXPECT_EQ(size_t(0), validator->errorCount());
+}
+
+TEST(CoverageValidator, invalidXmlIds)
+{
+    auto parser = libcellml::Parser::create();
+    auto model = parser->parseModel(fileContents("coverage/invalidxmlids.cellml"));
+
+    auto validator = libcellml::Validator::create();
+
+    validator->validateModel(model);
+    EXPECT_EQ(size_t(1), validator->errorCount());
```

Comment:
Which ids in `invalidxmlids.cellml` are invalid? All of them? Only one of them? If so, which one?

Reply:
[528a80f](https://github.com/cellml/libcellml/pull/1128/commits/528a80f1273ba2e874ef174e26b25907d50a9688)